app/main.py

```diff
@@ -1,10 +1,23 @@
 def copy_file(command: str) -> None:
+    # command = cp file.txt file-cope.txt
+
     file_name = command.split()
-    if len(file_name) == 3:
-        if file_name[0] == "cp" and file_name[1] != file_name[2]:
-            with (
-                open(file_name, "r") as file_in,
-                open(file_name, "w") as file_out
-            ):
-                file_to_read = file_in.read()
-                file_out.write(file_to_read)
+    if len(file_name) != 3:
+        return "Command has exactly 3 parts!"
+
+    _, main_file, destination_file = file_name
+
+    if main_file == destination_file:
+        return "Files are the same!"
+
+    try:
+        with (
+            open(main_file, "r") as file_in,
+            open(destination_file, "w") as file_out
+        ):
+            file_out.write(file_in.read())
+        return "Files were copied successfully!"
+    except FileNotFoundError as e:
```

Comment:
it's entirely reasonable to let the exception raise  and potentially terminate the program. no need to catch it and just print the message